plugin/storage/kafka/options_test.go

```diff
@@ -28,12 +30,18 @@ func TestOptionsWithFlags(t *testing.T) {
 	command.ParseFlags([]string{
 		"--kafka.producer.topic=topic1",
 		"--kafka.producer.brokers=127.0.0.1:9092, 0.0.0:1234",
-		"--kafka.producer.encoding=protobuf"})
+		"--kafka.producer.encoding=protobuf",
+		"--kafka.producer.required-acks=local",
+		"--kafka.producer.compression=gzip",
+		"--kafka.producer.compression.level=6"})
```

Comment:
This should have been changed to `compression-level` - so not sure why the unit tests seems to be passing atm?

Reply:
Such a stupid mistake from my side. :(
fixed and pushed it.